Assets/Scripts/LoadTerrain.cs

```diff
@@ -9,15 +9,8 @@ public class LoadTerrain : MonoBehaviour
 {
 
     //public Session session;
-    public GameObject prefab_distractor;
```

Comment:
aw yiss, net negative commits are the best!  :D

(i.e. more lines deleted than added)